pkg/workloads/helpers.go

```diff
@@ -188,6 +217,22 @@ func CortexMount() kcore.VolumeMount {
 	return k8s.EmptyDirVolumeMount(_cortexDirVolumeName, _cortexDirMountPath)
 }
 
+func JobSpecMount(name string) kcore.VolumeMount {
```

Comment:
Does it work to mount the entire config map (will it use the key for each file name)?

```go
func ConfigMapMount(name string) kcore.VolumeMount {
	return kcore.VolumeMount{
		Name:      name,
		MountPath: _cortexDirMountPath,
	}
}
```

And if that doesn't work (i.e. it clobbers anything that was there), it might still be good to do

```go
func ConfigMapMount(name string) kcore.VolumeMount {
	return kcore.VolumeMount{
		Name:      name,
		MountPath: path.Join(_cortexDirMountPath, "spec"),
	}
}
```